utils/slack.py

```diff
@@ -8,67 +8,73 @@
 from slack_sdk import WebClient
```

Comment:
it's under utils not under skills, because it's sending alerts for the deployments right?